packages/openmrs/src/http.js

```diff
@@ -0,0 +1,43 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * State object
+ * @typedef {Object} OpenMRSOptions
+ * @property {object} query - An object of query parameters to be encoded into the URL
+ * @property {object} headers - An object of all request headers
+ * @property {object} body - The request body (as JSON)
+ * @property {string} baseUrl - The base url for the request
+ */
+
+/**
+ * Make a HTTP request to any OpenMRS endpoint
+ * @example
+ * request("GET","http://msf-ocg-openmrs3-dev.westeurope.cloudapp.azure.com/openmrs/ws/rest/v1/patient/d3f7e1a8-0114-4de6-914b-41a11fc8a1a8", {
```

Comment:
:exclamation: Important! We do not want to allow absolute URLs here. Users should set state.configuration.instanceUrl

I actually think we should throw if an absolute URL is passed

Am I right in thinking that the common adaptor's request will throw if the url is absolute and baseURL has a value?

Reply:
Common does not throw an error, but rather `commcare` flags with a `404` if both the `instanceUrl` and `path` are absolute urls